resources/.eslintrc-import.js

```diff
@@ -35,16 +35,17 @@ module.exports = {
         "import/unambiguous": "error",                                     // Report potentially ambiguous parse goal (script vs. module)
 
         /* IMPORT: Style guide */
+        "import/exports-last": "off",                                           // Ensure all exports appear after other statements
```

Comment:
Decided against turning this one. [Have a look](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/exports-last.md) I think its probably a good idea but not sure it should be mandatory